src/com_tjnotifications/admin/models/notification.php

```diff
@@ -427,8 +427,8 @@ public function save($data)
 			$this->deleteBackendConfigs($backendConfigIdsToBeDeleted);
 
 			// 2.3 Common data for saving
-			$createdOn = !empty($data['created_on']) ? $data['created_on'] : '';
-			$updatedOn = !empty($data['updated_on']) ? $data['updated_on'] : '';
+			$createdOn = !empty($data['created_on']) ? $data['created_on'] : '0000-00-00 00:00:00';
```

Comment:
@pranotiTechjoomla for Joomla 4 default value for `created_on` column is null and not `0000-00-00 00:00:00`